### PR TITLE
adds ploomber-core recipe

### PR DESCRIPTION
**Pull request by edublancas**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by edublancas** [comment]
@conda-forge/help-python ready for review!

File: recipes/ploomber-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+test:
+  imports:
+    - ploomber_core
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/ploomber-core/
```
Comment (awvwgk) [inline recipes/ploomber-core/meta.yaml:36]:
```suggestion
  home: https://pypi.org/project/ploomber-core/
  dev_url: https://github.com/ploomber/core
```

File: recipes/ploomber-core/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - python >=3.6
+    - pyyaml
+
+test:
+  imports:
+    - ploomber_core
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (awvwgk) [inline recipes/ploomber-core/meta.yaml:33]: Can we run the unit tests?

```suggestion
  commands:
    - pip check
    - pytest tests/
  requires:
    - pip
    - pytest
  source_files:
    - tests/
```

Comment (edublancas) [reply recipes/ploomber-core/meta.yaml:33]: we run the tests on the main [repo](https://github.com/ploomber/core/actions), I thought of skipping them here to keep runtime short

Comment (edublancas) [reply recipes/ploomber-core/meta.yaml:33]: Hi @conda-forge/staged-recipes, any feedback here? Can we proceed with the existing recipe? We're running the test suite already in our CI system, so we can skip it here and save some build time.

File: recipes/ploomber-core/LICENSE [hunk 3]
```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```
Comment (carterbox) [inline recipes/ploomber-core/LICENSE:1]: Please delete this file. It's already included in the source tar.

File: recipes/ploomber-core/meta.yaml (hunk 2 above)
Comment (carterbox) [reply recipes/ploomber-core/meta.yaml:33]: Unit tests are not necessary in this case. Just packaging tests are sufficient.

**Review by carterbox — COMMENTED** [review]
LGTM after extra file deleted.

**Comment by edublancas** [comment]
done, license file deleted! @carterbox

**Pull request merged**